app/models/amt/AMTConditionTable.scala

```diff
@@ -28,6 +33,29 @@ class AMTConditionTable(tag: Tag) extends Table[AMTCondition](tag, Some("sidewal
 object AMTConditionTable {
   val db = play.api.db.slick.DB
   val amtConditions = TableQuery[AMTConditionTable]
+  val amtAssignments = TableQuery[AMTAssignmentTable]
+  val max_num_condition_assignments = 5
+
+  def getVolunteerIdByConditionId(amtConditionId: Int): String = db.withTransaction { implicit session =>
```

Comment:
This should probably be an Option[String] since the amtConditionId might not be in the database.